crates/typst/src/math/style.rs

```diff
@@ -382,6 +383,9 @@ pub fn styled_char(styles: StyleChain, c: char) -> char {
             (Cal | Frak | Mono | Bb, _, _) => return c,
         },
 
+        // Hebrew.
+        'א'..='ד' => 0x2135,
```

Comment:
could you escape the hebrew characters with `\u{..}` notation? otherwise, e.g. GitHub fails pretty badly at rendering the line in a readable way.